src/SourceBuild/content/eng/pipelines/templates/steps/publish-test-results-pr.yml

```diff
@@ -0,0 +1,207 @@
+parameters:
+- name: pipelineId
+  type: string
+
+- name: pullRequestTitle
+  type: string
+  default: Update Test Baselines 
+
+- name: pullRequestTargetBranch
+  type: string
+  default: $(Build.SourceBranchName)
+
+- name: testResultsPath
+  type: string
+
+steps:
+  - script: |
```

Comment:
I agree that this would work much better as a separate script. I looked into this when implementing these changes, and my only concern was that there is no `eng/pipeline/scripts` directory in the Arcade infra (as far as I can tell). I think this means that the script would just get overridden by the arcade infra. I suppose I could add the script at `src/SourceBuild/content/eng/common` if that's the case?

Reply:
I don't think Arcade is related to this issue. I was thinking you could create a new directory at `src/SourceBuild/content/eng/pipelines/scripts`.